jctools-core/src/main/java/org/jctools/queues/BaseMpscLinkedArrayQueue.java

```diff
@@ -131,15 +185,6 @@ public BaseMpscLinkedArrayQueue(int initialCapacity)
         soProducerLimit(mask); // we know it's all empty to start with
     }
 
-    /**
-     * This method assumes index is actually (index << 1) because lower bit is used for resize. This is
-     * compensated for by reducing the element shift. The computation is constant folded, so there's no cost.
-     */
-    private static long modifiedCalcElementOffset(long index, long mask)
```

Comment:
This was moved into a util method, LinkedArrayQueueUtil.modifiedCalcElementOffset